tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -97,7 +97,15 @@
 
   <mksrf_furban>
     <entry>
-      <data_filename>lnd/clm2/rawdata/mksrf_urban_0.05x0.05_simyr2000.c220127.nc</data_filename>
+      <data_filename>/glade/p/cgd/tss/people/oleson/Dynamic_Urban_Data/GaoOneil/PCTURB_GRID/Historical/urban_properties_GaoOneil_05deg_ThreeClass_2000_c20220910.nc</data_filename>
```

Comment:
@ekluzek 's comment
Change to use %y also for the time-slice/baseline file to be consistent with the requested year.

Reply:
Assigning to @slevisconsulting

Reply:
Sam also needs to copy the new files to lnd/clm2/rawdata and run `rimport`

Reply:
`./rimport -list urban_list.txt` in progress